README.md

```diff
@@ -88,6 +88,23 @@ This table shows all available options in `StompConfig`
 | onDebugMessage: Function(String)             | Function to be called for debug messages generated by the internal message handler                         |
 
 
+## Use Stomp with SockJS
+Use StompConfig.SockJS constructor instead of default StompConfig constructor.
+
+The url must be start with http/https instead of ws/wss.
+
+The StompClient url become automatically a ws url with SOCKJs pattern url like wss://host:port/path/xxxx/yyyy/websocket
```

Comment:
I think this line can be removed or should expand on what xxxx and yyyy are. This probably goes into to much protocol specific detail, so i suggest removing it.